src/app/shared/ui/form-result-view/form-result-view.component.ts

```diff
@@ -0,0 +1,34 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+export type FormResultInfo = {
+  messageHeader: string;
+  messageCallToAction?: string;
+};
+
+const formResultInfoDefault: FormResultInfo = {
+  messageHeader: '',
+};
+
+@Component({
+  selector: 'app-form-result-view',
```

Comment:
Czy tutaj nie byłoby może lepiej zmienić selector `app-form-result-view` na `app-form-result-view[formResultInfo]` żeby mieć pewność, że Input będzie przekazany?

Reply:
Zastanawiałem się dokładnie nad takim rozwiązaniem. Pytanie tylko co dla ciebie będzie wyglądało lepiej. To, że będziesz miała error, że nie można znaleźć takiego selectora jak ktoś nie poda Inputa i będzie się zastanawiał, gdzie popełnił błąd, czy bez tego i wtedy jeśli nie poda Inputa to wyświetli się mu pusty komunikat bez żadnego tekstu.

Reply:
Dodałem zmiany zgodnie z twoją sugestią. Komponent można będzie utworzyć tylko jeśli został podany input [formResultInfo].